README.md

```diff
@@ -1,63 +1,40 @@
 # scale-ci-ansible
 
 The repo contains playbooks for various scale-ci automation tasks.  Those include install,
-scaling, upgrade, and post install configuration. The environments under test include RHCOS and
-OCP on OSP installs.
+scaling, upgrade, and post install configuration. The environments under test include RHCOS 
+based OCP on AWS and OCP on OSP installs.
 
-# RHCOS Usage
+# RHCOS based OCP 4.X Usage
 
-The RHCOS playbooks focus on RHCOS on AWS at this time and are found in the `OCP-4.X` directory.
+The playbooks focus on RHCOS based OCP on AWS at this time and are found in the `OCP-4.X` directory.
 
 The prerequisites are simple and simply an orchestration machine with kubeconfig and oc command.
 The orchestration host can be localhost if properly setup.
 
-## RHCOS Post Install
-
-`OCP-4.X/post-install.yml` playbook is used to perform post RHCOS 4 cluster install operations.
-
-Running from CLI:
+### Prepare the Jump host
+- The jump host is the node which orchestrates the ocp install and configures the nodes to support the Scale-CI pipeline.
+- Jump host needs to be a RHEL box and preferred if it is based out of the AMI built by the image provisioner.
 
+### Run
+Clone the github repo:
 ```
-$ cp OCP-4.X/inventory.example inventory
-$ # Add orchestration host to inventory
-$ # Edit vars in OCP-4.X/vars/post-install.yml or define Environment vars (See below)
-$ time ansible-playbook -vv -i inventory OCP-4.X/post-install.yml
+$ git clone https://github.com/redhat-performance/scale-ci-ansible.git
+$ cd scale-ci-ansible/OCP-4.X
```

Comment:
In order to use the ansible.cfg, the end user would only have to cd into scale-ci-ansible and run `ansible-playbook -i inventory OCP-4.X/cluster.yml`